src/models/instrumentModel.ts

```diff
@@ -25,6 +25,14 @@ class InstrumentModel {
     return undefined;
   }
 
+  @computed
+  get invertedDisplayName() {
+    if (this.baseAsset && this.quoteAsset) {
+      return join('/', [this.quoteAsset.name, this.baseAsset.name]);
+    }
+    return undefined;
```

Comment:
Redundant, function will return undefined by default

Reply:
Deny, all code paths should return a string or undefined according to tslint suggestion.